src/utils/telegram.ts

```diff
@@ -96,15 +96,23 @@ class TmaSdk {
     useHaptic(type);
   }
 
+  // private onTouchEnd(event: ): void {
   private onTouchEnd(event: TouchEvent): void {
+    console.info('Touch end event detected');
     const clickableSelectors = 'button, a, [data-clickable], .el-button, .clickable, [role="button"]';
-    const clickedElement = event.target as Nullable<HTMLElement>;
+    let clickedElement = event.target as Nullable<HTMLElement>;
+    console.info('Touch end event on element:', clickedElement);
 
-    // Check if the clicked element matches any of the clickable selectors
-    if (clickedElement?.matches(clickableSelectors)) {
-      // Trigger Telegram WebApp HapticFeedback
-      useHaptic('soft');
+    while (clickedElement) {
```

Comment:
what will happen if we have
```
<s-button>
  <el-button>
    <button>
```
where everything is clickable? Looks like 3 haptics at the same time...